frontend/src/app/item/listagem-page-item/listagem-page-item.component.ts

```diff
@@ -0,0 +1,146 @@
+import { ItemService } from './../../services/item.service';
+import { Component, OnInit, ViewChild, ElementRef } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { PageNotificationService } from '@nuvem/primeng-components';
+import { finalize } from 'rxjs/operators';
+
+@Component({
+  selector: 'app-listagem-page-item',
+  templateUrl: './listagem-page-item.component.html',
+  styleUrls: ['./listagem-page-item.component.css']
+})
+export class ListagemPageItemComponent implements OnInit {
+
+    @ViewChild('inputFile', {static:false}) inputFile: ElementRef;
+    itens: any[] = [];
+    categorias: any[] = [];
```

Comment:
criar um categoriaModel, ou categoria alguma coisa, e tipa esse categoria com o que voce criou, any não é tipo, se esse categoria for um dropdown ou alguma coisa assim use o SelectItem